src/main/java/io/airbyte/testingtool/scenario/action/connection/ActionCreateConnection.java

```diff
@@ -34,6 +51,43 @@ public String getActionName() {
   @Override
   public void doActionInternal() throws ApiException {
     createConnection();
+    if (useNormalization) {
+      updateConnectionWithNormalization(connectionInstance.getConnectionId());
+    }
+  }
+
+  public void updateConnectionWithNormalization(UUID connectionId) throws ApiException {
```

Comment:
Please move this code into `setNormalizationFlag` into `ConnectionInstance` and call it if the destination `isSupportNormalization()` is `true`

Reply:
The input parameter will be the flag value. You can get it from the method `getNormalizationFlag()` (see comments below)

Reply:
updated it